man/systemd.socket.xml

```diff
@@ -935,6 +935,36 @@
         <xi:include href="version-info.xml" xpointer="v255"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DeferTrigger=</varname></term>
+
+        <listitem><para>Takes a boolean argument. May only be used when <varname>Accept=no</varname>.
+        If enabled, job mode <literal>meek</literal> instead of <literal>replace</literal> is used when
+        triggering the service, which means currently activating/running units that conflict with the service
+        won't be disturbed/brought down. Furthermore, if a conflict exists, the socket unit will wait for
+        current job queue to complete and potentially defer the activation by then. An upper limit of
+        total time to wait can be configured via <varname>DeferTriggerMaxSec=</varname>. The socket unit
+        will fail if all jobs have finished or the timeout has been reached but the conflict remains.
```

Comment:
The condition `if all jobs have finished` should be dropped, and always wait for the max sec.